docs/snippets/src/main.c

```diff
@@ -61,7 +61,7 @@ void * memory_ptr = NULL;
 //! [part4]
 // Get input port for model with one input
 ov_output_const_port_t* input_port = NULL;
-ov_model_const_input(model, &input_port);
+ov_compiled_model_input(compiled_model, &input_port);
```

Comment:
Is there any reason to change to be `ov_compiled_model_input`? I supposed they should get the same result, right?

Reply:
@riverlijunjie Thanks for code review! For your question, just match context.

<img width="1693" alt="Screenshot 2024-07-30 at 22 06 43" src="https://github.com/user-attachments/assets/45d35a0e-d18c-47fe-afb0-6d90d9bd4874">
<img width="1693" alt="Screenshot 2024-07-30 at 22 08 41" src="https://github.com/user-attachments/assets/4994aea0-e6ca-4a79-a8b4-ae7c7de646aa">

Reply:
It makes sense, thank your clarification!

Reply:
Please update description for this PR, thanks!

Reply:
> Please update description for this PR, thanks!

Done.